frontend/src/components/Public.vue

```diff
@@ -0,0 +1,345 @@
+<template>
+<div style="padding: 0px; margin: 0px;width: 100%; height: 100%;">
+    <div class="titleBar">
+        <ul class="titleContent">
+            <dropdown />
+            <li class="title">RPI Shuttle Tracker</li>
+        </ul>
+        <div v-if="$store.state.online" class="livebox">
```

Comment:
The "live" box in the current version looks nicer, so if we could keep the style from that, it would give it a more finished look.